src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,209 @@
+  // const [handleFilterButton, sethandleFilterButton] = useState('none');
+  const [products, setProducts] = useState([]);
+
+  // 필터 값 넣어두는 바구니
+  const [selectedTypes, setSelectedTypes] = useState({
+    whisky: false,
+    vodka: false,
+    cognac: false,
+    beer: false,
+    wine: false,
+    champagne: false,
+  });
+
+  //selectedCountry
+
+  const [filterButtonClick, setfilterButtonClick] = useState([]);
+
+  // const onClick = e => {
+  //   e.preventDefault(e);
+  //   handleFilterButton === 'none'
+  //     ? sethandleFilterButton('none')
+  //     : sethandleFilterButton('color');
+  // };
+
+  const postFilter = () => {
+    // selectedTypes에서 value가 true인 키를 다 꺼내, 그 키들을 패치에 넘겨
+    fetch(``)
+      .then(res => res.json())
+      .then(data => setProducts(data));
+    // 필터 되서 온 데이터를 products에 다시 담아서
+  };
+
+  /*
+  //주종 필터링
+  useEffect(() => {
+    fetch('http://10.58.6.41:8000/products')
+      .then(res => res.json())
+      .then(data => setProducts(data));
+    // .catch(e => console.log('error', e));
+  }, []);
+  */
+
+  // 국가 필터링
+  useEffect(() => {
+    fetch('./data/countries.json')
+      .then(res => res.json())
+      .then(data => setCountries(data));
+  }, []);
+  // 푸드 페어링 필터링
+  useEffect(() => {
+    fetch('./data/foodPairing.json')
+      .then(res => res.json())
+      .then(data => setFoodPairings(data));
+  }, []);
+
+  // 최초 데이터 요청
+  useEffect(() => {
+    fetch('/data/FilterList.json')
+      .then(res => res.json())
+      .then(data => setFilterButtons(data));
+  }, []);
+
+  // 필터링 버튼 눌렀을 때 필터링 요청
+  // const aaa = e => {
+  //   e.preventDefault();
+
+  //   fetch('http://10.58.6.41:8000/products', {
+  //     method: 'POST',
+  //     body: JSON.stringify({
+  //       type: e.target.textContent.toLowerCase(),
+  //     }),
+  //   })
+  //     .then(res => res.json())
+  //     .then(res => setProducts(res));
+
+  //   console.log(e.target.textContent.toLowerCase());
+  // };
+
+  return (
+    <article className="productList">
+      <div className="wrapper">
+        <div className="list">
+          <h1 className="listCopy">
+            Showing 280 wines between ₩10,000 - ₩40,000 rated above 3.8 stars
+          </h1>
+          <span className="listSubCopy">From 1 regional wine style</span>
+          <button className="sort" type="button">
+            리뷰
+          </button>
+          <div className="buttonResultLayout">
+            {filterButtonClick.map(e => {
+              return;
+              <div key={e.id}>{e.id}</div>;
+            })}
+          </div>
+        </div>
+        <div className="allLayout">
+          <div className="cardLayout">
+            <div className="filter">
+              <div className="filterTitle">
+                <span className="title">Types</span>
+                <span>select mutiple</span>
+              </div>
+              <form className="filterList">
+                {filterButtons &&
+                  filterButtons.map(filterButton => (
```

Comment:
filterButtons 컴포넌트의 초기값은 빈 배열로 truthy한 값이기 때문에, 단축평가의 조건은 항상 true입니다.
```suggestion
                {filterButtons.map(filterButton => (
```